app/assets/stylesheets/application.css

```diff
@@ -71,6 +72,8 @@ th h3, th p {
     background-color: #fff;
 }
 
+about.countainer
```

Comment:
Perfect! I'd be glad to help as well.  I'd start by putting the about logic back in the view,  ditching the extra layout file specifically for the about page and then we'll find a way to make a style specific page just for that page.  It should be pretty easy but fun!